src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1137,4 +1136,5 @@ public static void forceJVMLocaleToUSEnglish() {
         final List<T> sorted = values.stream().sorted().collect(Collectors.toList());
         return sorted.get(sorted.size() / 2);
     }
+
```

Comment:
Some changes appear in this file still

Reply:
Yes, I removed an unused import. And there's an extra blank line. Which one are you objecting to?